docs/getting-started.md

```diff
@@ -1,3 +1,52 @@
 # Getting started
 
-_TODO: Install opam, editor config, templates_
+Get up and running with Melange in no time!
+
+---
+
+## Install opam
+
+In order to work with Melange, you will have to install
+[opam](https://opam.ocaml.org/), a source-based package manager for OCaml.
+Instructions to install opam for the different operating systems can be found at
+https://opam.ocaml.org/doc/Install.html.
+
+## Template
```

Comment:
Unsure if we should also explain how to add Melange to an existing JavaScript project. @anmonteiro wdyt?